src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,26 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        String a = "";
+        switch (new Random().nextInt(6)) {
+            case 0:
+                a = "" + Colors.Red;
+                break;
+            case 1:
+                a = "" + Colors.Yellow;
+                break;
+            case 2:
+                a = "" + Colors.Green;
+                break;
+            case 3:
+                a = "" + Colors.Blue;
+                break;
+            default:
+                a = "" + Colors.White;
+        }
+        return a;
```

Comment:
we can treat enum as an array in some ways, instead of using switch/case we can get a random number (max number should be the length of our colors enum), then using this random index you can just access one of the colors from the enum and return it as a string